### ISSUE TITLE
FR: Apple Pencil eraser support. 

### ISSUE THREAD

**Issue by SeRealMF** [zsviczian/obsidian-excalidraw-plugin#1303]
**Is your feature request related to a problem? Please describe.**
I am using Excalidraw in Obsidian on my iPad and therefore I am using an Apple Pencil (2nd Generation). Usually, if you‘d double tap on the pencil it would switch to the eraser. This does not work for Excalidraw.

**Describe the solution you'd like**
Touble tap on Apple Pencil (2nd Gen) to select or deselect the eraser. 

**Describe alternatives you've considered**
Double tap on screen with x finger(s) to activate the eraser. Scribble to erase what was scribbled. 

**Additional context**
I saw you contributing so much to Excalidraw - thank you very much for that.

**Comment by Behv730** [comment]
agree, if it would work I could switch entirely obsidian for my university studies.
